src/pages/host/waiting-room/[code].page.tsx

```diff
@@ -79,7 +83,7 @@ const WaitingRoom = () => {
           <h1>Everyone Join!</h1>
           <h2 className="text-2xl font-normal">
             Room Code is:{" "}
-            <span className="block font-extrabold text-primary"> {code}</span>
+            <span className="text-primary block font-extrabold"> {code}</span>
```

Comment:
im guessing why this keeps changing is you dont have daisyUI classes in your tailwind extention or something bc it looks like its just the daisy classes that are changing order causing the code change.

Reply:
that makes sense. how do you have the daisy classes in ur extension?